### PR TITLE
Santu

### PR DESCRIPTION
**Pull request by Santugg2**
check branch

### CODE REVIEW

File: src/main/java/com/santhu/TestingBranch.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.santhu;
+
+public class TestingBranch {
+public static void main(String[] args) {
+	System.out.println("Checking branch");
```
Comment (kshivarama) [inline src/main/java/com/santhu/TestingBranch.java]: write some code and check

Comment (rsree20) [inline src/main/java/com/santhu/TestingBranch.java]: print your Name

File: src/main/java/com/santhu/TestingBranch.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.santhu;
+
+public class TestingBranch {
```
Comment (botlacharan) [inline src/main/java/com/santhu/TestingBranch.java]: change name

File: src/main/java/com/santhu/TestingBranch.java (hunk 1 above)
Comment (Santugg2) [reply src/main/java/com/santhu/TestingBranch.java]: ok dne

**Pull request merged**